tests/test_x509_ext.py

```diff
@@ -2220,16 +2275,20 @@ def test_other_name(self, backend):
         assert othernames == [expected]
 
     def test_certbuilder(self, backend):
-        sans = [u'*.example.org', u'*.\xf5\xe4\xf6\xfc.example.com',
-                u'foobar.example.net']
+        sans = [b'*.example.org', b'*.\xf5\xe4\xf6\xfc.example.com',
```

Comment:
This should probably be `b'*.xn--4ca7aey.example.com'` or just be removed from the SAN list?  Previously this tested that it was encoding idna properly, but now they're all just bytes being passed along (and we're passing non-ascii bytes here, so we're sort of testing whether OpenSSL will tolerate such garbage)